.kilocode/rules/memory-bank/tech.md

```
@@ -0,0 +1,229 @@
+- npm >= 8.0.0
+
+### Installation
+```bash
+npm install
+```
+
+### Running Development Server
+```bash
+npm run dev
+```
+Server runs on http://localhost:5173
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)